.eslintignore

```diff
@@ -1 +1,9 @@
 dist/*
+test/*
+webpack.config.js
+test.spec.js
+lib/crypto/bls.js
+lib/deterministicmnlist/SimplifiedMNList.js
```

Comment:
Why we ignore these files?

Reply:
Well, they have arrow functions, they have let/const usage etc... 
All things that are not part of the legacy code consensus of this repository, but they have been merged and it is not the work of this PR to fixed them all (they are many small issues).

Reply:
In my opinion, it's better to have consistency and linting rather than arrow functions in some files. But we have what we have :(